layouts/article/list-images.html

```diff
@@ -0,0 +1,65 @@
+                        {{ $currentSection := .CurrentSection }}
+                            {{ $i := 0 }}
+                            {{ $itemsPerRow := 4 }}
+                        {{ range and
+                            (where .Site.RegularPages.ByTitle "Section" .Section)
+                            (where .Pages ".Params.unlisted" "!=" "true")}}
+                            {{ if in (.RelPermalink | string) $currentSection.RelPermalink }}
+                                    <div class="col mb-4">
+                                        <div class="card">
+                                            <div class="card-body">
+                                                <h5 class="card-title"><a href="{{ .RelPermalink }}">{{ .Params.linktitle }}</a></h5>
+                                                <p class="card-text">{{ .Params.description }}</p>
+                                                <a href="{{ .RelPermalink }}image_specs/" class="card-link text-primary">Variants</a>
+                                                <a href="{{ .RelPermalink }}tags_history/" class="card-link text-primary">Tags</a>
+                                                <a href="{{ .RelPermalink }}tags_history/" class="card-link text-primary">Provenance</a>
+                                            </div>
+                                        </div>
+                                    </div>
+                                {{ $i = add $i 1 }}
+                                {{ if eq (mod $i $itemsPerRow) (sub 1 1) }}
+                                     <div class="w-100"></div>
+                                {{ end }}
+                            {{ end }}
+                        {{ end }}
+                        </div>
+                        </div>
+                    {{ else }}
+                        {{ range .Paginator.Pages }}
+                            <a class="docs-navigation-button docs-navigation-button-next" href="{{ .RelPermalink }}">
+                                <span class="next-title">{{ .Title }}</span>
+                                <div class="chevron-rotator">
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="10" height="15" viewBox="0 0 10 6" fill="none">
+                                        <path d="M1.06199 5.729L0.291992 4.938L4.99999 0.229004L9.70799 4.938L8.93799 5.729L4.99999 1.792L1.06199 5.729Z" fill="currentColor"/>
+                                    </svg>
+                                </div>
+                            </a>
+                            {{ if .Description }}
+                                <div class="button-description">{{ .Description }}</div>
+                            {{ end }}
+                        {{ end }}
+                        {{ template "_internal/pagination.html" . }}
+                    {{ end }}
+                </div>
+            </article>
+        </div>
+    </div>
+</div>

```

Comment:
Let me try to explain within the limited features of GitHub comments!
1) `col-md-12 col-lg-10 col-xl-8` made reflow weird, removing it gives a nice progressive reflow, up to you!
2) `row g-5 mb-5` applies gutters to everything without setting it on individual cards
3) `card flex-fill` makes all cards of a row take full height
4) `card-title mt-0` resets top padding from h5 to get even y padding in the card
5) The `justify-content-between` container makes your links appear as a justified row pinned to the bottom of the card
6) The `w-100` div forced a break every 4 elements, which translates to 4 per row on high res but then 3,1,3,1 as you resized the window, removing it allows it to flow naturally (4,4,4 then 3,3,3 then 2,2,2 then 1,1,1) and still caps at 4 but you may have had other reasons for it to exist (not 100% sure what the `eq sub 1 1` check did)

```suggestion
    <div class="row justify-content-center topic-container flex-shrink-1">
        <article>
            <h1 class="text-center mb-5">{{ if eq .CurrentSection .FirstSection }}{{ .Section | humanize }}{{ else }}{{ .Title }}{{ end }}</h1>
                {{ if eq .Params.type "article" }}
                    <div class="container">
                        <div class="row g-5 mb-5">
                        {{ $currentSection := .CurrentSection }}
                            {{ $i := 0 }}
                            {{ $itemsPerRow := 4 }}
                        {{ range and
                            (where .Site.RegularPages.ByTitle "Section" .Section)
                            (where .Pages ".Params.unlisted" "!=" "true")}}
                            {{ if in (.RelPermalink | string) $currentSection.RelPermalink }}
                                    <div class="col d-flex">
                                        <div class="card flex-fill">
                                            <div class="card-body d-flex flex-column">
                                                <h5 class="card-title mt-0">
                                                    <a href="{{ .RelPermalink }}">{{ .Params.linktitle }}</a>
                                                </h5>
                                                <p class="card-text">{{ .Params.description }}</p>
                                                <div class="d-flex flex-fill align-items-end justify-content-between">
                                                    <a href="{{ .RelPermalink }}image_specs/" class="card-link text-primary">Variants</a>
                                                    <a href="{{ .RelPermalink }}tags_history/" class="card-link text-primary">Tags</a>
                                                    <a href="{{ .RelPermalink }}tags_history/" class="card-link text-primary">Provenance</a>
                                                </div>
                                            </div>
                                        </div>
                                    </div>
                            {{ end }}
                        {{ end }}
                        </div>
                    </div>
                {{ else }}
                    {{ range .Paginator.Pages }}
                        <a class="docs-navigation-button docs-navigation-button-next" href="{{ .RelPermalink }}">
                            <span class="next-title">{{ .Title }}</span>
                            <div class="chevron-rotator">
                                <svg xmlns="http://www.w3.org/2000/svg" width="10" height="15" viewBox="0 0 10 6" fill="none">
                                    <path d="M1.06199 5.729L0.291992 4.938L4.99999 0.229004L9.70799 4.938L8.93799 5.729L4.99999 1.792L1.06199 5.729Z" fill="currentColor"/>
                                </svg>
                            </div>
                        </a>
                        {{ if .Description }}
                            <div class="button-description">{{ .Description }}</div>
                        {{ end }}
                    {{ end }}
                    {{ template "_internal/pagination.html" . }}
                {{ end }}
            </div>
        </article>
    </div>
</div>
```